libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -779,7 +779,7 @@ class LibCLCRemangler : public ASTConsumer {
   void Initialize(ASTContext &C) override {
     ASTCtx = &C;
     SMDiagnostic Err;
-#if SPIRV_ENABLE_OPAQUE_POINTERS
+#if defined(SPIRV_ENABLE_OPAQUE_POINTERS) || !defined(__SPIR__)
```

Comment:
Based on other uses of `SPIRV_ENABLE_OPAQUE_POINTERS` (including the old usage here) it seems to intend to also check the value of it, i.e. it should stay without a `defined()`:

```suggestion
#if SPIRV_ENABLE_OPAQUE_POINTERS || !defined(__SPIR__)
```

Reply:
Good catch thanks